_episodes/03-using-packages-and-channels.md

```diff
@@ -304,18 +213,16 @@ $ which pip
 > Conda functionality when mixing packages installed with Conda and Pip.
 {: .callout}
 
-If you find yourself needing to install a Python package that is only available via PyPI, then you should use the
-
+If you find yourself needing to install a Python package that is only available via PyPI, then you should use the copy of
 `pip`, which is installed automatically when you create a Conda environment with Python, to install the desired package
 from PyPI. Using the `pip` installed in your Conda environment to install Python packages not available via Conda
 channels will help you avoid difficult to debug issues that frequently arise when using Python packages installed via a
 `pip` that was not installed inside you Conda environment.
 
 > ## Conda (+Pip)
 >
-> Pitfalls of using Conda and `pip` together can be avoided by adopting the following practices.
->
-> * Always be sure your desired environment is *active* before installing anything using `pip`.
+> Pitfalls of using Conda and `pip` together can be avoided by 
+> always ensuring your **desired environment is active** before installing anything using `pip`.

```

Comment:
Do you think it is worth reminding people how to check which environment is active?

If so...

```suggestion
> always ensuring your **desired environment is active** before installing anything using `pip`. This can be done by looking at the output of `conda info`.

```